.github/workflows/release.yml

```diff
@@ -8,34 +8,48 @@ on:
 jobs:
   release:
     runs-on: ubuntu-latest
-    permissions:
-      contents: write
-      id-token: write
-      issues: write
-      repository-projects: write
-      deployments: write
-      packages: write
-      pull-requests: write
```

Comment:
I added these lines for "package provenance". Removing them _might_ make that break.

Reply:
Ah good call. These got wiped migrating over to the new pnpm workflow.